pyrogram/types/bots_and_keyboards/callback_query.py

```diff
@@ -110,7 +110,7 @@ async def _parse(client, callback_query, users) -> "CallbackQuery":
         # ignoring/replacing errors, this way, button clicks will still work.
         try:
             data = callback_query.data.decode()
-        except UnicodeDecodeError:
+        except (UnicodeDecodeError, AttributeError):
```

Comment:
`AttributeError`, what for?

Reply:
Game button will return `data` with `None`